xml/System.Drawing/Point.xml

```diff
@@ -84,7 +84,7 @@
         <AssemblyVersion>4.1.0.0</AssemblyVersion>
       </AssemblyInfo>
       <Docs>
-        <summary>Initializes a new instance of the <see cref="T:System.Drawing.Point" /> class with the specified coordinates.</summary>
+        <summary>Initializes a new instance of the <see cref="T:System.Drawing.Point" /> structure with the specified coordinates.</summary>
```

Comment:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Drawing.Point" /> struct with the specified coordinates.</summary>
```

Reply:
I think structure would have been fine, but am OK with using the C# keyword instead, it’s a common abbreviation even when talking.

Reply:
Yes, you're probably right, but since the page title uses "Struct", I suggested that.